client/config/misc.ts

```diff
@@ -1,3 +1,4 @@
 export const MAX_MOBILE_WIDTH = 767;
+export const MAX_MOBILE_HEIGHT = 600;
```

Comment:
Mobile screen-sizes are getting bigger and bigger, I did some research about screen-sizes and their height seems to vary between 800 - 920.

Reply:
Actually, I definitely agree with you.

Because with  `MAX_MOBILE_HEIGHT = 600;` , the breadcrumb appears or disappears depending on whether the software keyboard is displayed or not on many smartphones, such as iPhone 13 PRO and Google Pixel 6 pro. 
Although It have not been confirmed, I personally think this behavior is probably not what the designer intended.

However, since the `600px` might have been proposed for accessibility or other reasons, I would like to raise the proposal in the original issue that we would like to adopt `900px` for the reasons mentioned above, and then after gaining consensus, I will push the proposed viewport height.

Sincerely thanks for your thorough investigation and suggestion.

Reply:
We have not received any concern about the `900px` thresh so I tried to push the one to receive the review with updated branch.